cms/templates/emails/utilities_bulk_update_done_message.txt

```diff
@@ -0,0 +1,14 @@
+<%! from django.utils.translation import ugettext as _ %>
+${_("Hello, {user}").format(user=user_username)}
+
+% if success:
+${_("Your bulk update on the {course} course has successfully completed. Previously published problems have been automatically republished.").format(course=course_name)}
+% else:
+${_("Your bulk update on the {course} course has failed due to internal error. Problems in your course may be partially updated. Please contact our team for help.").format(course=course_name)}
+% endif
+
+${_("The following settings were modified:")}
```

Comment:
It looks like users will see this message even if the import fails.
It seems odd to tell them that we modified settings if there was really a failure.

Should we move this into the `success` block?
Or at the least, we should update the message.

Reply:
True, I've moved it to success only